client/my-sites/plugins/plugins-page-view-tracker/index.jsx

```diff
@@ -0,0 +1,18 @@
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+
+const PluginsPageViewTracker = ( { category, selectedSiteId, trackPageViews } ) => {
+	const analyticsPageTitle = 'Plugin Browser' + category ? ` > ${ category }` : '';
+	let analyticsPath = category ? `/plugins/browse/${ category }` : '/plugins';
```

Comment:
Can we delete this file and inline the base `<PageViewTracker>` components in each page component with paths specific to each page? Splitting the pages up was supposed to reduce this kind of ternary handling of all the different contexts.